packages/react/src/VideoPlayer/index.ts

```diff
@@ -1 +1,2 @@
 export * from './VideoPlayer'
+export * from './components'
```

Comment:
Can we keep these as private components please? If we do this, these sub components will be surfaced in the root of the package without namespacing [because of this](https://github.com/primer/brand/blob/main/packages/react/src/index.ts#L35). If we need to expose certain components, it should be via the VideoPlayer Object.

Reply:
Oversight when I switched to namespacing them, good catch thanks

4f99158cb9e13673c7c8aecd345ab8a41b2f284d